### PR TITLE
Add resolution presets (4:3, 16:10 etc) and other minor fixes/improvements

### PR DESCRIPTION
**Pull request by rsn8887**
See first commit message.

### CODE REVIEW

**Comment by rsn8887** [comment]
I added resolution presets (4:3, 16:10, etc.).

The resolution window size is saved, so you might have to enlarge it manually once to see the new buttons. Or just delete ux0:data/imgui.ini to reset the window size.

Summary of changes in this PR:
- ensure correct texture_size is sent to shaders by disabling SDL_Fullscreen. Together with the shader fixes in sdl12_gl, this now makes the LCD and sharp-bilinear (scanlines) masks appear correctly. (more info in 1st commit msg)

- fix build with latest imgui (rename of *_keys_usage(true) function to new name from latest imgui)

- fix wrong shader names `Sharp Bilinear (Scancode)` to `Sharp Bilinear Simple` and `Sharp Bilinear` to `Sharp Bilinear (Scanlines)`

- use `..\release\*` as argument to 7zip in Makefile, to include all necessary files in the 7zip release package with correct relative paths with just one wildcard.

- added preset buttons `4:3`, `16:10` etc. to the resolution window to allow easy selection of screen modes. Imgui updates the sliders automatically in real time with zero additional code required. It is amazing!

File: launcher/Makefile [hunk 1]
```diff
@@ -28,7 +28,7 @@ $(TARGET).vpk: $(TARGET).velf
 	cp -f param.sfo ../release/sce_sys/param.sfo
 	
 	#------------ Comment this if you don't have 7zip ------------------
-	7z a -tzip ./$(TARGET).vpk -r ..\release\sce_sys\* ..\release\eboot10.bin ..\release\eboot22.bin ..\release\eboot23.bin ..\release\eboot20.bin ..\release\eboot30.bin ..\release\eboot00.bin ..\release\eboot.bin 
+	7z a -tzip ./$(TARGET).vpk -r ..\release\*
```
Comment (Rinnegatamante) [inline launcher/Makefile:31]: Are you sure this works properly creating subdirectories?

File: source/sys_psp2.cpp [hunk 2]
```diff
@@ -73,11 +73,11 @@ void ImGui_callback() {
 					shader = SDL_SHADER_NONE;
 					SDL_SetVideoShader(SDL_SHADER_NONE);
 				}
-				if (ImGui::MenuItem("Sharp Bilinear", nullptr, shader == SDL_SHADER_SHARP_BILINEAR)){
+				if (ImGui::MenuItem("Sharp Bilinear (scanlines)", nullptr, shader == SDL_SHADER_SHARP_BILINEAR)){
 					shader = SDL_SHADER_SHARP_BILINEAR;
 					SDL_SetVideoShader(SDL_SHADER_SHARP_BILINEAR);
 				}
-				if (ImGui::MenuItem("Sharp Bilinear (Scancode)", nullptr, shader == SDL_SHADER_SHARP_BILINEAR_SIMPLE)){
+				if (ImGui::MenuItem("Sharp Bilinear Simple", nullptr, shader == SDL_SHADER_SHARP_BILINEAR_SIMPLE)){
```
Comment (Rinnegatamante) [inline source/sys_psp2.cpp]: I would prefer to call it just Sharp Bilinear

File: launcher/Makefile (hunk 1 above)
Comment (rsn8887) [reply launcher/Makefile:31]: Yes I tested it and it works.

File: source/sys_psp2.cpp (hunk 2 above)
Comment (rsn8887) [reply source/sys_psp2.cpp]: Ok done.

**Comment by rsn8887** [comment]
Should be ready to merge now

**Pull request closed without merging**